Dao/TagsDao.php

```diff
@@ -189,6 +189,21 @@ public function deleteContainerTag($idSite, $idContainerVersion, $idTag, $delete
         Db::query($query, $bind);
     }
 
+    /**
+     * @param int $idSite
+     * @param int $idContainerVersion
+     * @param string $deletedDate
+     */
+    public function deleteContainerTags($idSite, $idContainerVersion, $deletedDate)
```

Comment:
@ulcuber You aiming to update [Import.php](https://github.com/matomo-org/tag-manager/blob/4.x-dev/API/Import.php#L123) too with this PR right ?

Can you add those changes too ?

Reply:
I have tried to split it into small atomic independent issues. I'll optimize import next week

Reply:
@ulcuber Its better to include all such changes in 1 PR so that we can test and ensure that new changes do not lead to any regressions, I would suggest to add the Importer changes in same PR too or folk a branch from this and add those changes if you want to keep your changes separately